tests/integration-tests-with-streaming/dockerFullFlow-test.js

```diff
@@ -0,0 +1,265 @@
+    this.timeout(40000);
+    const jobPlatform = process.env.JOB_PLATFORM;
+    if (jobPlatform.toUpperCase() === 'DOCKER') {
+        describe('DOCKER', () => {
+            before(async () => {
+                await schedulerRequestCreator.init();
+                await testsRequestCreator.init();
+                await configRequestCreator.init();
+                await reportsRequestCreator.init();
+
+                kafkaHelper = new KafkaHelper(process.env.KAFKA_TOPIC, process.env.KAFKA_BROKERS.split(','));
+                await kafkaHelper.init();
+                await kafkaHelper.startConsuming();
+
+                await configRequestCreator.updateConfig({
+                    runner_docker_image: RUNNER_DOCKER_IMAGE
+                });
+
+                statsTime = Date.now();
+                testBody = require('../testExamples/Basic_test');
+                const response = await testsRequestCreator.createTest(testBody, {});
+                should(response.statusCode).eql(201);
+                testId = response.body.id;
+            });
+            after(async () => {
+                let containers = await docker.listContainers();
+                containers = containers.filter(container => {
+                    return container.Names && container.Names[0] && container.Names[0].includes('predator.');
+                });
+                containers.forEach(async container => {
+                    const containerToKill = await docker.getContainer(container.Id);
+                    await containerToKill.kill();
+                });
+                await kafkaHelper.disconnect();
+            });
+
+            describe('Good requests', () => {
+                describe('Run one time job with job-created and job-finished events published', () => {
+                    let createJobResponse;
+                    const runnerId = uuid.v4();
+
+                    it('Create the job', async () => {
+                        const validBody = {
+                            test_id: testId,
+                            type: 'load_test',
+                            arrival_rate: 2,
+                            duration: 1,
+                            run_immediately: true,
+                            notes: 'streaming notes'
+                        };
+
+                        createJobResponse = await schedulerRequestCreator.createJob(validBody, {
+                            'Content-Type': 'application/json'
+                        });
+
+                        jobId = createJobResponse.body.id;
+                        should(createJobResponse.status).eql(201);
+                    });
+
+                    it('Consume published event job-created', async () => {
+                        const jobCreatedMessage = await kafkaHelper.getLastMsgs(1);
+                        const valuePublished = JSON.parse(jobCreatedMessage[0].value);
+
+                        should(valuePublished.event).eql('job-created');
```

Comment:
i would verify other fields as well like you did in finished event

Reply:
done